src/views/GovernanceReviewTeam/GRBReview/PresentationLinksCard.tsx

```diff
@@ -0,0 +1,254 @@
+    eventEl?.addEventListener('click', linkHandler);
+    return () => {
+      eventEl?.removeEventListener('click', linkHandler);
+    };
+  }, []);
+
+  // Render empty if not an admin and no links
+  if (!isITGovAdmin && isEmpty) return null;
+
+  return (
+    <>
+      {/* Asynchronous presentation links card */}
+      <div className="usa-card__container margin-left-0 border-width-1px shadow-2 margin-top-3 margin-bottom-4">
+        <CardHeader>
+          <h3 className="display-inline-block margin-right-2 margin-bottom-0">
+            {t('asyncPresentation.title')}
+          </h3>
+        </CardHeader>
+        <CardBody>
+          {isEmpty ? (
+            <>
+              <Alert type="info" slim className="margin-bottom-1">
+                {t('asyncPresentation.adminEmptyAlert')}
+              </Alert>
+              <div className="margin-top-2 margin-bottom-neg-2">
+                <IconLink
+                  icon={<Icon.Add className="margin-right-1" />}
+                  to={`/it-governance/${systemIntakeID}/grb-review/presentation-links`}
+                >
+                  {t('asyncPresentation.addAsynchronousPresentationLinks')}
+                </IconLink>
+              </div>
+            </>
+          ) : (
+            <div className="margin-top-neg-1">
+              <UswdsReactLink
+                className="margin-right-2"
+                to={`/it-governance/${systemIntakeID}/grb-review/presentation-links`}
+              >
+                {t('asyncPresentation.editPresentationLinks')}
+              </UswdsReactLink>
+              <Button
+                type="button"
+                unstyled
+                className="text-error"
+                onClick={() => setRemovePresentationLinksModalOpen(true)}
+              >
+                {t('asyncPresentation.removeAllPresentationLinks')}
+              </Button>
+            </div>
+          )}
+        </CardBody>
+        <CardFooter>
+          {!isEmpty && (
+            <div
+              ref={externalModalScopeRef}
+              className="presentation-card-links display-flex flex-wrap border-top-1px border-gray-10 padding-top-2"
+            >
+              {recordingLink && (
+                <Link
+                  className="margin-right-2 display-flex flex-align-center"
+                  href={recordingLink}
+                  target="_blank"
+                >
+                  {t('asyncPresentation.viewRecording')}
+                  <Icon.Launch className="margin-left-05" />
+                </Link>
+              )}
+              {recordingPasscode && (
+                <span className="text-base margin-right-2">
+                  {t('asyncPresentation.passcode', {
+                    passcode: recordingPasscode
+                  })}
+                </span>
+              )}
+              {transcriptFileStatus === SystemIntakeDocumentStatus.AVAILABLE &&
+                transcriptFileURL && (
+                  <Link
```

Comment:
Might check with Zoe on this, but I'm pretty sure the recording passcode and transcript link should only display if there is a recording link.

Reply:
I just had a call with Zoe to go over something on the presentation links form so I asked her about this. If there is either a recording passcode or transcript but no recording link, display "No recording link available" in place of the link.